src/compounds/legacy-product-table/src/index.tsx

```diff
@@ -9,6 +9,128 @@ import CirclePercentageBar from '@uswitch/trustyle.circular-percentage-bar'
 
 import RowWrapper from './row-wrapper'
 
+interface WireFrameCellProps extends React.HTMLAttributes<HTMLDivElement> {
+  styles?: any
+}
+
+const WireFrameCell: React.FC<WireFrameCellProps> = ({ styles }) => {
+  const defaultStyles = {
+    backgroundColor: 'grey-20',
+    display: 'block'
+  }
+
+  return <span sx={{ ...defaultStyles, ...styles }}></span>
+}
+
+interface WireFrameProps extends React.HTMLAttributes<HTMLDivElement> {
+  type: 'header' | 'additionalInfo' | 'representativeExample'
+}
+
+const WireFrame: React.FC<WireFrameProps> = ({ type }) => {
+  const containerStyles: {
+    header: object
+    additionalInfo: object
+    representativeExample: object
+  } = {
+    header: {
+      paddingLeft: ['15px', '160px'],
+      paddingRight: ['15px', 0],
+      paddingY: '15px',
+      marginY: 0
+    },
+    additionalInfo: {
+      margin: ['15px', '15px 0']
+    },
+    representativeExample: {
+      marginTop: '5px',
+      padding: ['5px 20px 10px', '5px 40px 10px'],
+      backgroundColor: 'grey-10'
+    }
+  }
+
+  const defaultStyles: {
+    header: object
+    additionalInfo: object
+    representativeExample: object
+  } = {
+    header: {
+      width: ['320px', '425px'],
+      height: '14px'
+    },
+    additionalInfo: {
+      height: '10px',
+      width: '100%',
+      maxWidth: ['326px', '653px'],
```

Comment:
these numbers feel a bit magic, where do they come from, can we get anything from the theme?

Reply:
Turned them into percentages according to the design